paasta_tools/autoscaling/autoscaling_cluster_lib.py

```diff
@@ -1165,7 +1166,8 @@ def get_all_utilization_errors(
     return errors
 
 
-def autoscale_local_cluster(
+@to_blocking
```

Comment:
You can only `await` inside an `async def` (it's a syntax error otherwise). I feel like this is a little cleaner than using `block()` several times inside the function.

Reply:
and what is the reason this function must be blocking? can't we make the caller use asyncio to block on it?

Reply:
At some point, something needs to be synchronous. In this particular case, this function is just called by autoscale_cluster.py's main, so I can just clean this up and call it with `block` or start an event loop there.

In an earlier iteration of this branch, I had several other methods that were decorated with `@to_blocking`, since it's a convenient way to avoid changing the interface of functions that are called from several places. I've since cleaned those up, so I'll go ahead and clean this up too.

Reply:
I still suspect this would get annoying -- that certain parts of the codebase (particularly mesos_tools) will be filled with lots of `await something_async()`. It certainly would make bugs where you're not awaiting a a _async function really obvious when reading the code, but I'm not sure it's worth the extra characters.

(Honestly, I think I'm just dreading that mass rename.)

Reply:
yeah, mass rename is dreadful. let's merge it as is and figure out and apply the naming convention iteratively?